Antidote/ActiveCallViewController.m

```diff
@@ -319,6 +339,84 @@ - (void)showCallPausedByFriend
     [self createFriendAvatar];
 }
 
+- (void)setVideoView:(UIView *)videoView
+{
+
+    if (_videoView == videoView) {
+        return;
+    }
+
+    if (_videoView) {
+        [_videoView removeFromSuperview];
+    }
+
+    _videoView = videoView;
+
+    if (! videoView) {
+
+        if (! self.previewLayer) {
+            [self updateCallsControlViewExpand];
+        }
+
+        return;
+    }
+
+
+    if (self.previewView) {
+        [self.view insertSubview:self.videoView belowSubview:self.previewView];
+    }
+    else {
+        [self.view addSubview:self.videoView];
```

Comment:
Also a nice feature to have - on tapping on `videoContainer` hide/show all controls (i.e. enter full screen mode)